vunit/modelsim_interface.py

```diff
@@ -276,7 +276,8 @@ def _create_load_function(self,  # pylint: disable=too-many-arguments,too-many-l
         else:
             coverage_file = join(output_path, "coverage.ucdb")
             self._coverage_files.add(coverage_file)
-            coverage_save_cmd = "coverage save -onexit -assert -directive -cvg -codeAll {%s}" % fix_path(coverage_file)
+            coverage_save_cmd = "coverage save -onexit -assert -directive -cvg -codeAll {%s}" \
+			% (os.path.basename(os.path.dirname(output_path)), fix_path(coverage_file))
```

Comment:
Yes but it is not obvious from the code. As I said I am missing a comment in the code or intermediate variable to make it obvious that it is the test name that is extracted.

In the future the output path migth not even contain the test name since we migth need to compress it due to Windows path length limitations. Then this code would have to be modified, by us, or you will notice it.